package-lock.json

```diff
@@ -15,7 +15,7 @@
 				"@feathersjs/errors": "^4.5.11",
 				"@feathersjs/express": "^4.5.11",
 				"@feathersjs/feathers": "^4.5.11",
```

Comment:
I think featherejs/socketio comes together with feathers update, consisting of multiple packages
Not sure this is the correct way to upgrade. The upgrade was suggested by dependabot, and all the dependencies seem to resolve, but if you look all the feathers packages are at the same version 5.0.8